app/services/bulk_upload/lettings/year2024/row_parser.rb

```diff
@@ -1135,7 +1145,7 @@ def attributes_for_log
     attributes["renewal"] = renewal
     attributes["scheme"] = scheme
     attributes["location"] = location
-    attributes["assigned_to"] = assigned_to || bulk_upload.user
+    attributes["assigned_to"] = bulk_upload.user.support? ? nil : (assigned_to || bulk_upload.user)
```

Comment:
Should this be `assigned_to || (bulk_upload.user.support? ? nil : bulk_upload.user)`?

Reply:
Changed